descwl_coadd/coadd.py

```diff
@@ -802,13 +861,15 @@ def _finish_init(self):
             jacobian=jac,
             psf=psf_obs,
             store_pixels=False,
+            mfrac=mfrac,
         )
 
         flags_for_maskfrac = self.coadd_exp.mask.getPlaneBitMask('BRIGHT')
-        self.meta['mask_frac'] = get_masked_frac(
+        self.meta['bright_frac'] = get_masked_frac(
```

Comment:
need to make sure this rename is ok downstream

Reply:
Do you want to run a quick test or maybe just change the downstream code? If you point to where the change needs to go, I can make it.

Reply:
you are also saving bright_frac so I can adjust downstream if needed